src/components/navbar/index.jsx

```diff
@@ -0,0 +1,55 @@
+"use client";
```

Comment:
Why is this a client component?

Reply:
Because usePathname can only be used using use client